pkg/cmd/target/history.go

```diff
@@ -0,0 +1,170 @@
+/*
+SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and Gardener contributors
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package target
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/gardener/gardenctl-v2/internal/util"
+	"github.com/gardener/gardenctl-v2/pkg/cmd/base"
+	"github.com/gardener/gardenctl-v2/pkg/target"
+)
+
+const (
+	historyFile = "history"
+)
+
+// NewCmdHistory returns a new target history command
+func NewCmdHistory(f util.Factory, o *HistoryOptions) *cobra.Command {
```

Comment:
please change signature to
`func NewCmdHistory(f util.Factory, ioStreams util.IOStreams) *cobra.Command {`

we still have a few other commands like `func NewCmdUnset(f util.Factory, o *UnsetOptions) *cobra.Command {` that follow the pattern where the options is passed but we want to get rid of it.

Reply:
Thanks for the information, didn't know that. 7214aae will take a looked in the future to get rid of it